examples/src/main/java/org/apache/spark/examples/ml/JavaOneVsRestExample.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.examples.ml;
+
+import org.apache.commons.cli.*;
```

Comment:
Yeah, I suppose I couldn't recommend removing the flags now that they're here, or rewriting this without the library just to remove the dependency (it's just a test scope dependency and already comes in, evidently). Let's add it in https://github.com/apache/spark/pull/7697 instead, which is also using it. There, it might make more sense to curb the use of flags and make it more of an example; the issue here is that this example doesn't show me how to run it -- just says to set the options to something appropriate.